src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+
+    public Figure randomFigure() {
+        int figureIndex = (int) (Math.random() * 5);
+
+        switch (figureIndex) {
+            case 1: Figure randomSquare = new Square("square",
+                    ColorSupplier.randomColor(), (int) (Math.random() * 16));
```

Comment:
5, 8 and 16 are magic numbers - please create constants

Reply:
done